src/main/java/flaxbeard/steamcraft/SteamcraftBook.java

```diff
@@ -433,10 +489,49 @@ public static void registerExosuit() {
             if (Config.enableJumpAssist) {
                 SteamcraftRegistry.addResearch("research.JumpAssist.name", "!research.ExoHeel.name", new BookPageItem("research.JumpAssist.name", "research.JumpAssist.0", true, new ItemStack(SteamcraftItems.jumpAssist)), new BookPageCrafting("", "jumpAssist1", "jumpAssist2"));
             }
+            if (Config.enableAnchorHeels) {
+                if (Config.enableLeadPlate && OreDictionary.getOres("ingotLead").size() > 0 &&
+                  !Config.enableAnchorAnvilRecipe) {
+                    SteamcraftRegistry.addResearch(
+                      "research.AnchorHeels.name",
+                      "!research.ExoHeel.name",
+                      new BookPageItem("research.AnchorHeels.name",
+                        "research.AnchorHeelsLead.0", true,
+                        new ItemStack(SteamcraftItems.anchorHeels)),
+                      new BookPageCrafting("", "anchorHeels")
+                    );
+                } else {
+                    SteamcraftRegistry.addResearch(
+                      "research.AnchorHeels.name",
+                      "!research.ExoHeel.name",
+                      new BookPageItem("research.AnchorHeels.name",
+                        "research.AnchorHeelsIron.0", true,
+                        new ItemStack(SteamcraftItems.anchorHeels)),
+                      new BookPageCrafting("", "anchorHeels")
+                    );
+                }
+            }
```

Comment:
There are essentially the same, I would have a boolean local variable defined as `Config.enableLeadPlate && OreDictionary.getOres("ingotLead").size() > 0 && !Config.enableAnchorAnvilRecipe` and then use a ternary to get whether it should be "research.AnchorHeelsIron.0" or "AnchorHeelsLead", you could even just insert the Iron/Lead part and have "research.AnchorHeels<insert>.0" common